### PR TITLE
Renable cc_deployment_updater and fix readiness check

### PR DESCRIPTION
**Pull request by andrew-edgar**
Add back the cc_deployment updater to the scheduler pod.

## Description
Basically this reverted commit dfcc97d441efab31ec49a07b3cf87062baf7aeac.  But also fixed the readiness check

## Motivation and Context
This will allow the cc_deployment_updater to run and reenable the zero downtime functionality in CF.

## How Has This Been Tested?
Deployed this in a full kubecf env and ran the CATs tests which validate this functionality

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by jandubois — APPROVED** [review]

**Comment by jandubois** [comment]
@andrew-edgar Did you test the new readiness probe? It always fails for me, most likely because the server only listens on the 127.0.0.1 address, but k8s will probe the actual pod IP address: https://github.com/cloudfoundry/capi-k8s-release/issues/91

So I think this PR can only be merged after we have bumped cf-deployment to something more recent.

**Review by jandubois — DISMISSED** [review]
I've rebased this PR on top of latest `master` to get cf-deployment-16.9.0, which should listen on `0.0.0.0` for the readiness probe.

Tentatively approving the PR for running CI, but want confirmation from @andrew-edgar that the rolling updates work properly with the updated PR before merging.

**Comment by andrew-edgar** [comment]
I had tested the readiness probe and it did work for me.  but maybe some of our other changes caused that to work.  picking up 16.9.0 seems the right thing if that helps.  excellent.

**Comment by andrew-edgar** [comment]
@jandubois yes we are now seeing in the older version the readiness check failing.  I think we will get this fixed like you say when we bump to the latest cf-deployment.  thanks

**Review by jandubois — APPROVED** [review]

**Review by jandubois — DISMISSED** [review]

**Review by jandubois — APPROVED** [review]
I've rebased this PR on top of latest `master`, which dismissed the previous approval. Approving again...

**Pull request merged**
